src/EventDispatcher/SimpleEventDispatcher.php

```diff
@@ -5,6 +5,7 @@
 namespace Yiisoft\Test\Support\EventDispatcher;
 
 use Closure;
+use PHPUnit\Framework\Assert;
```

Comment:
We need to get rid of this dependency. You can throw custom exception instead

Reply:
Is this package meant to be used without PhpUnit?

Reply:
As you can see, PHPUnit is not declared in the `requre` section of `composer.json`.
PHPUnit required in dev only for self-testing.

Reply:
Custom exception is not what should be used in tests. If this package really meant to be used without PhpUnit, feel free to close this PR.